test/ProtocolFees.t.sol

```diff
@@ -79,7 +84,6 @@ contract ProtocolFeesTest is BoostTest {
         );
         uint256 boostId2 = boost.nextBoostId();
         vm.prank(owner);
-        snapStart("CreateBoostWithProtocolFee");
```

Comment:
@Orland0x I think this overwrote the other "CreateBoostWithProtocolFee" and I suspect the massive fee overhead to come from here

Reply:
ah i see, there was 2 snapshots for the same thing before. the increase is because its now tracking the cost of creating the first boost ever, which is more expensive than latter ones.

Reply:
i moved the snapshot https://github.com/snapshot-labs/boost/pull/33/commits/7bbc16984d2467922be32322777b743ace52af32